lib/features/family/features/reflect/presentation/pages/summary_screen.dart

```diff
@@ -79,13 +81,13 @@ class _SummaryScreenState extends State<SummaryScreen> {
                   ),
                 ],
               ),
+              //hmm
+
               const Spacer(),
               FunButton(
                 onTap: () {
-                  Navigator.of(context).popUntil(
-                    ModalRoute.withName(
-                      FamilyPages.profileSelection.name,
-                    ),
```

Comment:
Previous solution was clearing the stack first, is that not needed anymore?

Reply:
Somewhere else in the app we also have a clear solution for this navigation (with a for loop I think)

Reply:
Ah okay! Maybe that was with a push what I have seen :)

Did a search on Google: With go() "Navigating to a destination in GoRouter will replace the current stack of screens with the screens configured to be displayed for the destination route."

Reply:
I guess that is the same as named :)

Reply:
> Navigating to a destination in GoRouter will replace the current stack of screens with the screens configured to be displayed for the destination route. 
[GoRouter docs](https://pub.dev/documentation/go_router/latest/topics/Navigation-topic.html)